### ISSUE TITLE
Missing relative path support on Windows?

### ISSUE THREAD

**Issue by TobiasForner** [Cimbali/pympress#189]
### Environment
- Windows 10
- Python 3.9.0 (output of python --version, pympress log file says 3.8.2)

### Pympress
- 1.5.3
- Installation method: chocolatey

### Expected behaviour
When using `pympress myfile.pdf` the file is loaded as a presentation without error

### Steps to reproduce
1. navigate to file location in command line
2. execute `pympress myfile.pdf` 

### Problem
the presentation is not shown, instead there is an error:
Could not find the file "C:\Program Files\VideoLAN\VLC\myfile.pdf"

What is reported in pympress.log?

```
WARNING:pympress.extras:Building FileWatcher
ERROR:pympress.extras:Video support using GStreamer is disabled.
INFO:pympress.extras:Caused by ValueError: Namespace Gst not available
ERROR:pympress.extras:Video support using VLC is disabled.
INFO:pympress.extras:Caused by FileNotFoundError: Could not find module 'libvlc.dll' (or one of its dependencies). Try using the full path with constructor syntax.
INFO:__main__:Pympress: 1.5.3 ; Python: 3.8.2 ; OS: Windows 10 10.0.19041 ; Gtk 3.24.16 ; GLib 2.64.1 ; Poppler 0.87.0 cairo ; Cairo 1.16.0 , pycairo 1.19.1 ; Media: GtkImage gif player
WARNING:pympress.ui:Not starting content or presenter window full screen because there is only one monitor
```

Does the problem still happen if you remove your config file?
Yes.

**Comment by TobiasForner** [comment]
It is worth noting that everything works as expected when using an absolute path.

**Comment by Cimbali** [comment]
Hi @TobiasForner and thanks for reporting the issue.

Can you try it with the [1.6.0 beta](https://github.com/Cimbali/pympress/releases/tag/v1.6.0-beta6) ? The path management system has been changed to use URLs now, according to Gio semantics, and I can’t reproduce your issue (even with a space in the path as I s uspet that could be the cause of the error).

Also which command line are you using, `cmd` or PowerShell ?

**Comment by TobiasForner** [comment]
Hi, @Cimbali , thanks for your quick response!
The beta version works perfectly ( and there is indeed a space in the path).
I am using cmd, but via cmder which may add/change some commands.

**Comment by Cimbali** [comment]
All right, well glad it’s fixed then ? 1.6.0 should be out shortly.
